### PR TITLE
Update usability.md

### LINKED ISSUE #2425: USWDS-Site - Bug: Remove mention of Alert dismissability
### Describe the bug

The Alert page's usability guidance mentions Alerts should be dismissable.

### Steps to reproduce the bug

https://github.com/uswds/uswds-site/blob/7943e666c36de06b0cf7190cb860d89b25eda4d2/_components/alert/guidance/usability.md?plain=1#L5

### Expected Behavior

Should this be removed until there is guidance on _how_ to do this, given it is [not on the roadmap](https://github.com/uswds/uswds/issues/988#issuecomment-405390279)?

### Screenshots

_No response_

### System setup

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.

### PR DESCRIPTION
**Pull request by sarah-sch**
Updating alert guidance to explain that USWDS is working on a solution that will help developers make alert dismissible.

# Summary

Updating usability guidance on Alert page to explain that USWDS team is working on a solution to make Alerts dismissible.

## Related issue

Closes #2425 

## Preview link

Preview link: [Preview Alert page here.](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/ss-alert-page-update/components/alert/)

## Problem statement

Currently, USWDS guidance recommends making the Alert dismissible, but we don't provide an easy way for devs to implement a dismissible alert. This causes confusion for users.

## Solution

This PR updates our guidance to explain that we are working on a solution. We're also providing a link so that the community can follow our progress.

## Major changes

NA

## Testing and review

_Share recommended methods for reviewing this change._

### CODE REVIEW

File: _components/alert/guidance/usability.md [hunk 1]
```diff
@@ -2,5 +2,5 @@
 - **Be polite.** Be polite in error messages — don’t blame the user.
 - **Alerts are an opportunity.** Users will read a message that helps them resolve an error even if they generally won’t read documentation; include some educational material in your error message.
 - **Don’t overdo it.** Too many notifications will either overwhelm or annoy the user and are likely to be ignored.
-- **Allow a user to dismiss a notification wherever appropriate.**
+- **Allow a user to dismiss a notification whenever appropriate.** USWDS is working on a way to help developers implement a dismissable alert. [You can follow our progress on GitHub]([url](https://github.com/uswds/uswds/issues/5754)).
```
Comment (mahoneycm) [inline _components/alert/guidance/usability.md:5]: Looks like the formatting got a little messed up here and broke the link.

This should resolve it:

```suggestion
- **Allow a user to dismiss a notification whenever appropriate.** USWDS is working on a way to help developers implement a dismissable alert. [You can follow our progress on GitHub](https://github.com/uswds/uswds/issues/5754).
```

File: _data/changelogs/component-alert.yml
Comment (amyleadem) [inline _data/changelogs/component-alert.yml:1]: Here is the changelog entry for this PR. Feel free to edit as needed!

**Review by amyleadem — COMMENTED** [review]
@sarah-sch This looks good! I think this change warrants a changelog entry, so I drafted one in `_data/changelogs/component-alert.yml` and pushed it up. Feel free to edit as needed!

**Comment by sarah-sch** [comment]
@mahoneycm - I fixed that link. Thanks! Can you approve if the page looks good?

**Comment by sarah-sch** [comment]
> @sarah-sch This looks good! I think this change warrants a changelog entry, so I drafted one in `_data/changelogs/component-alert.yml` and pushed it up. Feel free to edit as needed!

@amyleadem - thank you! The changelog entry looks good to me.

**Review by mahoneycm — APPROVED** [review]
lgtm!

**Comment by sarah-sch** [comment]
> @sarah-sch This looks good! I think this change warrants a changelog entry, so I drafted one in `_data/changelogs/component-alert.yml` and pushed it up. Feel free to edit as needed!

Thanks, @amyleadem! I fixed the spelling of "dismissible" here and in the new guidance, and I think it's ready for FFR now.

**Review by annepetersen — COMMENTED** [review]
LGTM

**Comment by mejiaj** [comment]
@annepetersen ready for override merge, only outstanding issues are the pa11y timeout failures.

**Pull request merged**